mpcs/benches/basefold.rs

```diff
@@ -143,10 +144,14 @@ fn bench_batch_commit_open_verify_goldilocks<Pcs: PolynomialCommitmentScheme<E>>
             let values: Vec<E> = evals
                 .iter()
                 .map(Evaluation::value)
-                .map(|x| *x)
+                .copied()
```

Comment:
That's an unrelated improvement, isn't it?  (It's small enough that we can leave it in this PR, however.)